packages/prepo-shared-contracts/test/SafeOwnableCaller.test.ts

```diff
@@ -0,0 +1,123 @@
+  }
+
+  const setupSafeOwnableCallerAndMockContract = async (): Promise<void> => {
+    await setupSafeOwnableCallerTest()
+    const mockOwnedContractFactory = await smock.mock('SafeOwnable')
+    mockOwnedContract = await mockOwnedContractFactory.deploy()
+  }
+
+  describe('initial state', () => {
+    beforeEach(async () => {
+      await setupSafeOwnableCallerTest()
+    })
+
+    it('sets owner to deployer', async () => {
+      expect(await safeOwnableCaller.owner()).to.eq(deployer.address)
+    })
+  })
+
+  describe('# transferOwnership', () => {
+    beforeEach(async () => {
+      await setupSafeOwnableCallerAndMockContract()
+      await mockOwnedContract.connect(owner).transferOwnership(safeOwnableCaller.address)
+      await safeOwnableCaller.connect(owner)['acceptOwnership(address)'](mockOwnedContract.address)
+    })
+
+    it('reverts if not owner', async () => {
+      expect(await safeOwnableCaller.owner()).to.not.eq(user1.address)
+
+      await expect(
+        safeOwnableCaller
+          .connect(user1)
+          ['transferOwnership(address,address)'](mockOwnedContract.address, user1.address)
+      ).revertedWith('Ownable: caller is not the owner')
+    })
+
+    it("reverts if owned contract's transferOwnership reverts", async () => {
+      mockOwnedContract.transferOwnership.reverts()
+
+      await expect(
+        safeOwnableCaller
+          .connect(owner)
+          ['transferOwnership(address,address)'](mockOwnedContract.address, user1.address)
+      ).to.be.reverted
+      expect(mockOwnedContract.transferOwnership).to.have.been.called
+    })
+
+    it("succeeds if owned contract's transferOwnership doesn't revert", async () => {
+      await safeOwnableCaller
+        .connect(owner)
+        ['transferOwnership(address,address)'](mockOwnedContract.address, user1.address)
+
+      expect(mockOwnedContract.transferOwnership).to.not.have.been.reverted
+      expect(mockOwnedContract.transferOwnership).to.have.been.called
+    })
+
+    it('calls transferOwnership of owned contract with correct parameters', async () => {
+      await safeOwnableCaller
+        .connect(owner)
+        ['transferOwnership(address,address)'](mockOwnedContract.address, user1.address)
+
+      expect(mockOwnedContract.transferOwnership).to.have.been.calledWith(user1.address)
+    })
+  })
+
+  describe('# acceptOwnership', () => {
+    beforeEach(async () => {
+      await setupSafeOwnableCallerAndMockContract()
+      await mockOwnedContract.connect(owner).transferOwnership(safeOwnableCaller.address)
+    })
+
+    it('reverts if not owner', async () => {
+      expect(await safeOwnableCaller.owner()).to.not.eq(user1.address)
+
+      await expect(
+        safeOwnableCaller.connect(user1)['acceptOwnership(address)'](mockOwnedContract.address)
+      ).revertedWith('Ownable: caller is not the owner')
+    })
+
+    it("reverts if owned contract's acceptOwnership reverts", async () => {
```

Comment:
```suggestion
    it("reverts if contract's acceptOwnership reverts", async () => {
```
No need for specifying owned contract, since any contract can be passed in and we'd still expect a revert to be passed upwards.